EasyPost.Tests.Net/ReportTest.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Net;
+using System.Threading;
```

Comment:
Nit: this can be removed since its not being used in anywhere.

Reply:
It will be when this gets merged with the async stuff, so I'll keep it for now.